sklearn/utils/class_weight.py

```diff
@@ -14,10 +14,11 @@
         "class_weight": [dict, StrOptions({"balanced"}), None],
         "classes": [np.ndarray],
         "y": ["array-like"],
+        "sample_weight": ["array-like", None],
     },
     prefer_skip_nested_validation=True,
 )
-def compute_class_weight(class_weight, *, classes, y):
+def compute_class_weight(class_weight, *, classes, y, sample_weight=None):
```

Comment:
Could you please add a new unit test case for `compute_class_weight` with `sample_weight != None` in `skearn/utils/tests/test_class_weight.py`?

Reply:
This test could implement the usual weight / repetition equivalence check for this function (with random integer weights, including a few 0).